bin/lib/askWhenConfigExists.js

```diff
@@ -0,0 +1,51 @@
+const { readFile } = require('fs/promises');
+const inquirer = require('inquirer');
+
+const isObject = x => typeof x === 'object' && !Array.isArray(x) && x !== null;
+
+// TODO: add this function to the toolbox
+const merge = (obj, obj2) =>
+  Object.entries(obj)
+    .concat(Object.entries(obj2))
+    .reduce((acc, [key, value]) => {
+      const accValue = acc[key];
+      return Array.isArray(accValue) && Array.isArray(value)
+        ? { ...acc, [key]: accValue.concat(value) }
+        : isObject(accValue) && isObject(value)
+        ? { ...acc, [key]: merge(accValue, value) }
+        : { ...acc, [key]: value };
+    }, {});
+
+const Choice = {
+  noConfig: 'noConfig',
+  overwrite: 'overwrite',
+  extend: 'extend',
+};
+
+const askWhenConfigExists = async eslintrc => {
+  const file = await readFile('.eslintrc.json').catch(() => null);
```

Comment:
What about configs in `.eslintrc`, `eslintrc.yml`, `eslintrc.yaml` and, `package.json`?